src/whylogs/core/metrics/nlp_metrics.py

```diff
@@ -0,0 +1,80 @@
+from typing import List, Union
+
+import jiwer
+
+from whylogs.core.statistics import NumberTracker
+from whylogs.proto import NLPMetricsMessage
+
+
+class NLPMetrics:
+    def __init__(self, prediction_field: str = None, target_field: str = None):
+        self.prediction_field = prediction_field
+        self.target_field = target_field
+
+    def update(self, predictions: Union[List[str], str], targets: Union[List[str]], transform=None) -> None:
+        """
+        Function adds predictions and targets computation of nlp metrics.
+
+        Args:
+            predictions (Union[str,List[str]]):
+            targets (Union[List[str],str]):
+
+        """
+        if transform:
+            mes = jiwer.compute_measures(truth=targets, hypothesis=predictions, truth_transform=transform, hypothesis_transform=transform)
+        else:
+            mes = jiwer.compute_measures(truth=targets, hypothesis=predictions)
+
+        self.mer.update(mes["mer"])
```

Comment:
Can we store these strings as constants?

Reply:
sure